### ISSUE TITLE
Visual bug when immersive engineering mod is present (1.20.1) (Forge)

### ISSUE THREAD

**Issue by tooopt** [ChloePrime/AAAParticles#18]
If AAAParticles and immersive engineering are installed, all the particles are rendered on top of everything. 

![2024-08-04_23 02 26](https://github.com/user-attachments/assets/171a2ddf-6d31-4fc6-923c-436aeea69cb6)

To reproduce
Install immersive engineering and AAAParticles
Set config "effek_lightning_in_production_environment" to "true"
Create a world and /summon minecraft:lightningbolt

Same visual bug happens with any other effek particles added by other mods if immersive engineering is present.

It also spams the log file with
[23:05:51] [Render thread/INFO] [mojang/GlDebug]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Depth formats do not match.'

**Comment by TheASEStefan** [comment]
So the particles appear randomly? Because I had the same exact issue and it hasn't been fixed yet, but I suggest using version 1.0.8 or maybe older. Those should work because for me they did.

**Comment by tooopt** [comment]
No, the particles are rendered on top of everything. If, let's say, a lightning bolt with this effek particle would strike somewhere, I would see it through all the blocks. 

Previous version have much more serious bugs when immersive engineering is installed, and render everything you hold like it is pitch black, and it continues to spam the log message I have previously mentioned, and then leads to a game crash. 

The ;atest version is better, because it spams inly when the particle itself is visible, but still not good

**Comment by TheASEStefan** [comment]
Hmm alright then, seems like a different issue, ignore my comment then!

**Comment by ChloePrime** [comment]
![image](https://github.com/user-attachments/assets/b031d0a8-0ef7-4bee-af31-60f0c3d70749)
Not reproduceable :(
